### ISSUE TITLE
Issue explained in a Video

### ISSUE THREAD

**Issue by usmaann1** [JuiceMind/JuiceMind#335]
https://www.loom.com/share/abc0f8c0a7c444b1a164dbcd30490d71

**Comment by juicemind-dev** [comment]
I don't fully understand what the issue is. Is it only that when you click in, it highlights rather than putting the cursor where you clicked?

**Comment by usmaann1** [comment]
@juicemind-dev If I click on the newly added row after a '/'  then the dropdown should open
am I right ?

**Comment by juicemind-dev** [comment]
It does in your video though

**Comment by usmaann1** [comment]
Yeah, It happens in my video but the issue is most of the time when I click then dropdown doesn't open

**Comment by juicemind-dev** [comment]
Yeah I think you're right. The video just isn't very good.
